doc/specs/#653 - Quake Mode/#653 - Quake Mode.md

```diff
@@ -0,0 +1,680 @@
+
+Else:
+* activate the one on this desktop (don't move it)
+</td>
+</tr>
+<!-- ----------------------------------------------------------------------- -->
+<tr>
+<td><code>"toCurrent"</code><br> Summon the MRU window TO the current monitor</td>
+<td>Go to the desktop the window is on, move to this monitor</td>
+<td>Move the window to this desktop, move to this monitor</td>
+<td>
+
+If there isn't one on this desktop:
+* create a new one (on this monitor)
+
+Else:
+* activate the one on this desktop, move to this window
+</td>
+</tr>
+<!-- ----------------------------------------------------------------------- -->
+<tr>
+<td><code>"onCurrent"</code><br> Summon the MRU window for the current monitor</td>
+<td>
+
+If there is a window on this monitor on any desktop,
+* Go to the desktop the window is on (leave position alone)
+
+else
+* Create a new window on this monitor  & desktop
+</td>
+<td>
+
+If there is a window on this monitor on any desktop,
+* Move the window to this desktop (leave position alone)
+
+else
+* Create a new window on this monitor  & desktop
+</td>
+<td>
+
+If there isn't one on this desktop, (even if there is one on this monitor on
+another desktop),
+* create a new one on this monitor
+
+Else if ( there is one on this desktop, not this monitor)
+* create a new one on this monitor
+
+Else (one on this desktop & monitor)
+* Activate the one on this desktop (don't move)
+</td>
+</tr>
+<!-- ----------------------------------------------------------------------- -->
+<tr>
+<td><code>int</code><br> Summon the MRU window for monitor N</td>
+<td>
+
+If there is a window on monitor N on any desktop,
+* Go to the desktop the window is on (leave position alone)
+
+else
+* Create a new window on this monitor  & desktop
+</td>
+<td>
+
+If there is a window on monitor N on any desktop,
+* Move the window to this desktop (leave position alone)
+
+else
+* Create a new window on this monitor & desktop
+</td>
+<td>
+
+If there isn't one on this desktop, (even if there is one on monitor N on
+another desktop),
+* create a new one on monitor N
+
+Else if ( there is one on this desktop, not monitor N)
+* create a new one on monitor N
+
+Else (one on this desktop & monitor N)
+* Activate the one on this desktop (don't move)
+</td>
+</tr>
+</table>
+
+
+##### Stories, revisited
+
+With the above settings, let's re-examine the original user stories, and see how
+they fit into the above settings. (_Stories that are omitted aren't relevant to
+the discussion of these settings_)
+
+> When the `desktop` param is omitted below, that can be interpreted as "any
+> `desktop` value will make sense here"
+
+* **Story A** Press a hotkey anywhere to activate the single Terminal window
+  wherever it was
+  - This is `{ "monitor": "any", "desktop": "any" }`
+* **Story B** Press a hotkey anywhere to activate the single Terminal window _on
+  the current monitor_. If it wasn't previously on that monitor, move it there.
+  - This is `{ "monitor": "toCurrent" }`

```

Comment:
I think I missed it above where you define what the default behavior is for `desktop` (and I guess `monitor`) if you specify one and not the other. Or at least I didn't realize it was possible until this example. Potentially make that more obvious.

Reply:
I added a blurb before the table to hopefully help clarify.